google_auth_oauthlib/helpers.py

```diff
@@ -133,6 +133,8 @@ def credentials_from_session(session, client_config=None):
             token_url=client_config.get("token_uri"),
             client_id=client_config.get("client_id"),
             client_secret=client_config.get("client_secret"),
+            token_info_url=client_config.get("token_info_url"),
+            scopes=session.scope,
```

Comment:
why is this being added?

Reply:
This ends up passing the token_info_url from the login config to the credential class, so we can have an easier time doing introspection later.  It also passes the scopes used to the credential class because it is useful for querying

Reply:
If a user wants to know whether their credentials have the necessary permissions before trying to use them, they will need to know what scopes were used to create the credentials

Reply:
Sorry I meant "why is this being added" for the scopes. I get why token_info_url is being added.